<launch>
    <rosparam command="load" file="$(env HOME)/icar-its-data/config/start.yaml" />

    <node pkg="ds4_driver" name="ds4" type="ds4_driver_node.py" output="log">
        <remap from="/status" to="/ds4_raw/to_pc" />
        <remap from="/set_feedback" to="/ds4_raw/from_pc" />
    </node>

    <node pkg="icar_miscellaneous" type="log" name="log" output="screen" />
    <node pkg="icar_routine" type="interface_stm32" name="interface_stm32" output="screen" />
    <node pkg="icar_routine" type="interface_ds4" name="interface_ds4" output="screen" />
    <node pkg="icar_routine" type="interface_px4" name="interface_px4" output="screen" />
    <node pkg="icar_routine" type="transform_broadcaster" name="transform_broadcaster" output="screen" />
    <node pkg="icar_routine" type="odometry" name="odometry" output="screen" />
    <node pkg="icar_routine" type="routine" name="routine" output="screen" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(env HOME)/icar-its-data/rviz/start.rviz" />
</launch>